examples/jsm/loaders/SVGLoader.js

```diff
@@ -661,7 +661,7 @@ class SVGLoader extends Loader {
 
 					// Remove empty rules
 					const definitions = Object.fromEntries(
-						Object.entries( stylesheet.style ).filter( ( [ _, v ] ) => v !== '' )
+						Object.entries( stylesheet.style ).filter( ( [ , v ] ) => v !== '' )
```

Comment:
Just `[ v ]` instead of `[ , v ]` seems more clear.

Reply:
Do you realize that it's the same right? I'm checking for an empty value, not a key.

![Screenshot 2022-01-14 at 8 11 10](https://user-images.githubusercontent.com/660716/149466404-ad3f704c-e5d3-4fcd-ad5a-16e648839ac0.png)
.